admin/meetings/urls.py

```diff
@@ -0,0 +1,12 @@
+from __future__ import absolute_import
+
+from django.conf.urls import url
+
+from admin.meetings import views
+
+urlpatterns = [
+    url(r'^$', views.MeetingListView.as_view(), name='list'),
+    url(r'^create/$', views.MeetingCreateFormView.as_view(), name='create'),
```

Comment:
This suggestion is a confluence of three requests: `GET: list of meetings`, `GET: blank meeting form`, `POST: new meeting`. Should I change the list to be on another URL?